tools/ci_build/github/linux/docker/scripts/install_manylinux2010.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -e
+set -x
+
+while getopts p: parameter_Option
+do case "${parameter_Option}"
+in
+p) PYTHON_VER=${OPTARG};;
+esac
+done
+
+PYTHON_VER=${PYTHON_VER:=3.5}
+CPYTHON_VER=cp${PYTHON_VER//./}
+
+curl -fsSLo /tmp/rpmforge.rpm http://repository.it4i.cz/mirrors/repoforge/redhat/el6/en/x86_64/rpmforge/RPMS/rpmforge-release-0.5.3-1.el6.rf.x86_64.rpm
```

Comment:
Please add a comment, which do you need from this repo?

Reply:
done